examples/prompting/Zero_shot_prompting.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
<p>Do we need this temp = 0 assignment? maybe drop?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, good catch. We are dropping the temperature parameter